### PR TITLE
NicheCompass sunburst plot 

### PR DESCRIPTION
**Pull request by AnnaMaguza**

### CODE REVIEW

File: visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb [hunk 1]
```diff
@@ -0,0 +1,1955 @@
+{
```
Comment (cartal) [inline visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb:1]: <p>Check date</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/4/discussion/#comment-1791658037'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (cartal) [reply visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb:1]: @AnnaMaguza could you please check the new version and see if there is something I am missing?

Comment (AnnaMaguza) [reply visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb:1]: <p>Looks good :) </p>

Comment (AnnaMaguza) [inline visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb:1]: <p>Why do you extract gene weights for 20 genes but in the previous cell, you extracted 60 genes? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/4/discussion/#comment-1791681223'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

Comment (cartal) [reply visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb:1]: <p>Because I forgot to change the cell 🙈</p>

Comment (cartal) [reply visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb:1]: Solved now, please check update.

**Pull request merged**